src/components/Videos.vue

```diff
@@ -129,6 +139,20 @@ export default {
 	},
 
 	methods: {
+		hideHeaderAndFooter(e){
+			// work arround to get the state of the fullscreen button, aria-selected attribute is not reliable
+			this.fullscreenButton = !this.fullscreenButton
+			if(this.fullscreenButton) {
```

Comment:
Can't we use a class for that ?

Reply:
Ah, indeed. Not comfortable having an app touch footer and main though. If there is an issue with those style, it is hard to track who set them.

Would there be any other way of fixing the issue?

Reply:
True, I'll try to research a better way

Reply:
Done. I took over for Hamza and refactored the styling to a class that is toggled on `main` and `footer`.